.github/workflows/ci.yaml

```diff
@@ -11,6 +11,28 @@ on:
       - "*"
 
 jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      # Adapted from https://github.com/d3adb5/helm-unittest-action/
+      - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+      - name: Install Helm
+        uses: azure/setup-helm@5119fcb9089d432beecbf79bb2c7915207344b78 # v3.5
+      - name: Install helm-unittest
+        run: helm plugin install https://github.com/helm-unittest/helm-unittest
+      - name: Set chart directories to test as environment variable
+        run: |
+          echo "CHARTS_TO_TEST=$(find . -type f -name 'Chart.yaml' -exec dirname {} \; | tr '\n' ' ')" >> $GITHUB_ENV
+        shell: bash
+      - name: Fetch chart dependencies
+        run: |
+          for chart in ${{ env.CHARTS_TO_TEST }}; do
+            helm dependency update "$chart" 
+          done
+        shell: bash
```

Comment:
This should not harm, but we don't have dependencies.

Reply:
we have the policy reporter: https://github.com/kubewarden/helm-charts/blob/3a3895054ce33b235a65d1e3a189ea9ed88f48cb/charts/kubewarden-controller/Chart.yaml#L49